packages/rest/src/__tests__/unit/request-body.validator.test.ts

```diff
@@ -333,7 +333,7 @@ describe('validateRequestBody', () => {
     it('nested $ref in reference', () => {
       const details: RestHttpErrors.ValidationErrorDetails[] = [
         {
-          path: '.accounts[0].address.city',
+          path: '/accounts/0/address/city',
```

Comment:
Is it introduced by `jsonPointers: true,`?

Reply:
Yes, it is because of `jsonPointers: true`